src/screens/home/components/subreddits/style.css

```diff
@@ -28,7 +28,7 @@
     background-color: #fff;
     border-radius: 10px;
     margin: 20px;
-    height: 2050px;
+    height: 2100px;
```

Comment:
Why did you do this?

Reply:
Because if i dont the subreddit catgeroies dont fit properly in their main component!